src/rpm_ostree/actor.rs

```diff
@@ -1,14 +1,25 @@
 //! rpm-ostree client actor.
 
+use super::cli_status::StatusJSON;
 use super::Release;
 use actix::prelude::*;
 use failure::Fallible;
+use filetime::FileTime;
 use log::trace;
 use std::collections::BTreeSet;
 
+/// Cache of local deployments.
+#[derive(Clone, Debug)]
+pub struct StatusCache {
+    pub status: StatusJSON,
+    pub mtime: FileTime,
+}
+
 /// Client actor for rpm-ostree.
 #[derive(Debug, Default, Clone)]
-pub struct RpmOstreeClient {}
+pub struct RpmOstreeClient {
+    pub status_cache: Option<StatusCache>,
```

Comment:
For reference, I believe that this works only because this actor is spawned on a single thread.
If we move to a n>1 threadpool, it will result in each actor-thread having its own cache.

There is no correctness issue in doing this and it isn't a real problem right now, so I think we can keep this. Perhaps we should add a quite note as a reminder, though.